ios-base/DroidKaigi 2020/Session/Detail/SessionDetailViewController.swift

```diff
@@ -0,0 +1,204 @@
+import ios_combined
+import Nuke
+import RxCocoa
+import RxSwift
+import UIKit
+
+final class SessionDetailViewController: UIViewController {
+    var session: Session!
```

Comment:
Why this property is not private?

Reply:
How to initialize this VC is not determined, so once I made it possible to set from outside.

Reply:
I see. This is OK 😄